src/app/components/TransactionOverview/TransactionOverview.tsx

```diff
@@ -14,73 +13,82 @@ export default function TransactionOverview({
   deleteTransaction,
   transactions,
 }: InitialStateType): JSX.Element {
-  const [showTransaction, setShowTransaction] = useState<boolean>(false)
-
-  function handleOnclick() {
-    setShowTransaction(!showTransaction)
-  }
-
   return (
     <div>
```

Comment:
Maybe a section or something else than a div?

Reply:
you are absolutely right